cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canola oil
```

Comment:
Looks like you have learned to spell "Canola" rightly. I have never heard this name before. People say its healthy.